packages/flutter/lib/src/widgets/scroll_position.dart

```diff
@@ -898,6 +914,15 @@ abstract class ScrollPosition extends ViewportOffset with ScrollMetrics {
     UserScrollNotification(metrics: copyWith(), context: context.notificationContext!, direction: direction).dispatch(context.notificationContext);
   }
 
+  /// Dispatches a notification that the contents metrics has changed.
+  ///
+  /// This notification is dispatched when the dimensions of the contents have
```

Comment:
I am not sure if I have expressed it clearly, 
it is likely that need your help to reorganize the language, thank you!